apps/box/src/components/Text.tsx

```diff
@@ -0,0 +1,10 @@
+import { FxText } from '@functionland/component-library';
+import React from 'react';
+import { TextProps } from 'react-native';
+
+export const HeaderText = (props: TextProps) => (
```

Comment:
this should be props for `FxText`.

We can export `FxTextProps` from text.tsx similar to what we are doing in box.tsx

Reply:
Good point. Initially I had put TextProps to prevent the user from editing the comp and then we discussed our design philosophy more and decided we would stick with the pattern of allowing overrides

Reply:
Have updated